### DISCUSSION TITLE
How to make pull requests to self by default?

### DISCUSSION THREAD

**Discussion by gqqnbig** [community/community#23803] [New to GitHub]
<p>My repository is a fork of an upstream repo. Whenever I make a pull request in my repo, GitHub by default chooses the upstream repo as the PR target. Is there a way for GitHub to by default choose my own repo as the PR target?</p>

**Comment by yamiacat** [comment, marked as answer]
<p>While your repository is a fork, pull requests will always default to the upstream repository, sorry!</p>
<p>If you don’t need to keep the fork relationship, GitHub Support can detach your repository and turn it into a stand alone, and then pull requests will default to self. (If it’s the fork of a private repository we’ll need authorization from an admin of the root repository.)</p>

**Comment by meganharvey27** [comment]
Detach

**Comment by Aaronmuloisi** [comment]
@67245402

**Comment by klondikemarlen** [comment]
See request for this feature at https://github.com/orgs/community/discussions/11729 and some workarounds.
